locales/de-DE/App.json

```diff
@@ -243,5 +238,7 @@
   "Label": "Etikett",
   "Last used": "Zuletzt benutzt",
   "Transactions": "Vorgänge",
-  "ALPH amount": "ALPH-Betrag"
+  "ALPH amount": "ALPH-Betrag",
+  "Set lock time": "Set lock time",
+  "Tweak gas settings": "Tweak gas settings"
```

Comment:
@Iix I could use your help with these 2 new sentences! You can just comment here and I'll add it. 🙏 Much obliged!

Reply:
"Set lock time": "Sperrdauer einstellen",
"Tweak gas settings": "Gas Einstellungen optimieren"